src/ItemGrid.cpp

```diff
@@ -440,6 +440,29 @@ int cItemGrid::RemoveItem(const cItem & a_ItemStack)
 
 
 
+cItem * cItemGrid::FindItem(const cItem & a_RecipeItem)
+{
+	if (!m_Slots.IsStorageAllocated())
+	{
+		return nullptr;
+	}
+
+	for (int i = 0; i < m_Slots.size(); i++)
+	{
+		if ((a_RecipeItem.m_ItemType == m_Slots[i].m_ItemType) &&
+			((a_RecipeItem.m_ItemDamage < 0) || (a_RecipeItem.m_ItemDamage == m_Slots[i].m_ItemDamage)))
+		{
```

Comment:
```suggestion
		if (
			(a_RecipeItem.m_ItemType == m_Slots[i].m_ItemType) &&
			((a_RecipeItem.m_ItemDamage < 0) || (a_RecipeItem.m_ItemDamage == m_Slots[i].m_ItemDamage))
		)
		{
```

Reply:
Or even better, use the `sItemCompare` somehow?

Reply:
Puh, ok, not sure if it gets more readable. At least the comparison is only at one place.